src/app/components/podcasts-section/podcasts-section.component.ts

```diff
@@ -13,15 +13,30 @@ import { Subscription } from 'rxjs/Subscription';
 export class PodcastsSectionComponent implements OnInit, OnDestroy {
   @Input() public podcastFeed: string;
   public podcasts: Podcast[];
+  public keyword: string;
   public podcastFeedSubscription: Subscription;
 
-  private feedUri: string = 'https://djvlwqlqta.execute-api.eu-west-1.amazonaws.com/Prod/';
+  // local
+  private feedUri = 'http://127.0.0.1:8080/';
```

Comment:
sanirim bunlari unutmussun burada. Buradan kaldirmak lazim

Reply:
yok unutmadım aslında, chat kanalında bahsettiğim konulardan biriydi bu. lambda manifest'in çalışacağından eminsek, prod linklerini yazıp PR geçecektim normalde.